test/acceptance/pages.py

```diff
@@ -15,6 +15,34 @@ class PageConfigurationError(Exception):
     pass
 
 
+class MultipleAssessmentPage(PageObject):
+    """
+    Page object for subsection unit containing multiple ORA problems
+    Does not inherit OpenAssessmentPage object
+    Used to test accessibility of multiple ORA problems on one page
+    """
+    def __init__(self, browser, problem_location):
+        """
+        Configure a page object for a particular ORA problem.
+
+        Args:
+            browser (Selenium browser): The browser object used by the tests.
+            problem_location (unicode): URL path for the problem, appended to the base URL.
+
+        """
+        super(MultipleAssessmentPage, self).__init__(browser)
+        self._problem_location = problem_location
+
+    def is_browser_on_page(self):
+        return self.q(css='#main').is_present()
```

Comment:
Add a comment why the css is #main (don't want to limit to a specific ORA problem).